doc/programming/language/modules.rst

```diff
@@ -79,10 +79,17 @@ A module may define the following global properties:
 
     ``%byte-order = ORDER;``
         Defaults the byte order for any parsing inside this module to
-        `<expr>`, where ``ORDER`` must be of typei is type
+        `<expr>`, where ``ORDER`` must be of type is type
         :ref:`spicy_ByteOrder`.
 
     ``%spicy-version = "VERSION";``
         Specifies that the module requires a given minimum version of
         Spicy, where ``VERSION`` must be a string of the form ``X.Y``
         or ``X.Y.Z``.
+
+    ``%skip = ( REGEXP | Null );``
+        Specifies a pattern which should be skipped when encountered in the input stream in between parsing of unit fields.
```

Comment:
```suggestion
        Specifies a pattern which should be skipped when encountered in the input stream in between parsing of unit fields (incl. before/after the first/last field).
```

Reply:
We can leave out the `Null` here: at the module-level that's not going to be useful (it's just restating the default).